src/test/java/org/folio/fqm/utils/ItemStatusEnumTest.java

```diff
@@ -0,0 +1,65 @@
+package org.folio.fqm.utils;
+
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.net.HttpURLConnection;
+import java.net.URL;
+
+class ItemStatusEnumTest {
+
+  @Test
+  void checkNumberOfItemStatuesDoNotChanged() {
+    try {
+
+      URL url = new URL("https://raw.githubusercontent.com/folio-org/mod-inventory-storage/master/ramls/item.json");
+
+      HttpURLConnection connection = (HttpURLConnection) url.openConnection();
+      connection.setRequestMethod("GET");
+
+      int responseCode = connection.getResponseCode();
+
+      if (responseCode == HttpURLConnection.HTTP_OK) {
+        BufferedReader in = new BufferedReader(new InputStreamReader(connection.getInputStream()));
+        String inputLine;
+        StringBuilder content = new StringBuilder();
+
+        while ((inputLine = in.readLine()) != null) {
+          content.append(inputLine);
+        }
```

Comment:
Whenever I see something like this, I wonder what it'd look like as a `for`-loop...

```java
        for (String inputLine = in.readLine(); inputLine != null; inputLine = in.readLine()) {
          content.append(inputLine);
        }
```

...that looks kind of gross. `while`-loop it is! 🙂

Reply:
Even better: use a stream! `BufferedReader.lines()` returns a `Stream<String>`, which can be collected directly into  a new string.

```suggestion
        String content = in.lines().collect(Collectors.joining());
```